packages/shared_preferences/shared_preferences_linux/pubspec.yaml

```diff
@@ -11,17 +11,17 @@ flutter:
         pluginClass: none
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.12.0-259.8.beta <3.0.0"
```

Comment:
Why such a specific version? Is there a precise dependency you need?

Reply:
Honestly it is the version the `dart migrate` tool put by default. No real reason to be honest.

Reply:
I've since learned it just puts whatever version of Dart it used in the migration. Since we've been using `-0` everywhere in the repo so far I'd like to standardize on that.

Reply:
So I just put 2.12.0-0 right ?

Reply:
Correct.